make/autoconf/flags-other.m4

```diff
@@ -88,6 +88,18 @@ AC_DEFUN([FLAGS_SETUP_RCFLAGS],
   AC_SUBST(RCFLAGS)
 ])
 
+AC_DEFUN([FLAGS_SETUP_NMFLAGS],
+[
+  # On AIX, we need to set NM flags
+  if test "x$OPENJDK_TARGET_OS" = xaix; then
+    NMFLAGS="-X64"
+  else
```

Comment:
The else case is not needed, as an empty NMFLAGS would be the default. See the handling of RCFLAGS above.

Reply:
I removed the needless else.